x/did/client/rest/rest.go

```diff
@@ -14,12 +14,8 @@ const (
 // RegisterRoutes registers did-related REST handlers to a router
 func RegisterRoutes(clientCtx client.Context, r *mux.Router) {
 	// this line is used by starport scaffolding # 2
-	registerQueryRoutes(clientCtx, r)
-	registerTxHandlers(clientCtx, r)
-
-	registerQueryRoutes(clientCtx, r)
-	registerTxHandlers(clientCtx, r)
-
+	/*registerQueryRoutes(clientCtx, r)
+	registerTxHandlers(clientCtx, r)*/
```

Comment:
Maybe legacy rest api.

Reply:
Yeah. It's okay to comment them out for now.

We may need to re-enable the legacy REST API. Medipass basically uses our Java SDK, but some insurance companies uses our REST API directly, IIRC.
If it's too difficult to enforce them to upgrade their logic, we may need to provide the legacy REST API with them for a while.
In that case, we also need to test whether the legacy API in Stargate is compatible with the previous one.